docs/changehistory/NextVersion.md

```diff
@@ -76,6 +79,10 @@ Reduced API surface of an `ElectronApp` class to only allow white-listed APIs fr
 
 The argument for [IModelHost.startup]($backend) has been changed from [IModelHostConfiguration]($backend) to the [IModelHostOptions]($backend) interface. This matches the approach on the frontend for [IModelApp.startup]($frontend) and makes it easier to supply startup options. `IModelHostConfiguration` implements `IModelHostOptions`, so existing code will continue to work without changes.
 
+## Tooling
+
+The `@itwin/core-webpack-tools` and `@itwin/backend-webpack-tools` packages have been updated to support [Webpack 5](https://webpack.js.org/) and now requires a peer dependency of _webpack@^5_. Please refer to their [changelog](https://github.com/webpack/changelog-v5/blob/master/README.md) and [migration guide](https://github.com/webpack/changelog-v5/blob/master/MIGRATION%20GUIDE.md) as you update. If you would like to continue using _webpack@^4_, please use versions ~3.2.0 of these packages.
```

Comment:
> If you would like to continue using _webpack@^4_, please use versions ~3.2.0 of these packages

"These packages" meaning "iTwin.js packages"? Do we really want to encourage people not to update?

Reply:
yes meaning the itwinjs pkgs, i will make that more explicit. 

Updating to webpack 5 is breaking change and imo these pkgs shouldve been broken out into their own version policy.
So it's not that we don't want users to update, it just might take them some effort and won't be as simple as just updating the pkg ver in the pkg.json. I could also remove this line altogether if you think thats best

Reply:
Removal seems preferable to me -- the change log is about what changed in 3.3, so it should be obvious to readers that if they don't like something that changed in 3.3, they can remain on 3.2.

Reply:
updated